integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/nextmove/AsicHandler.java

```diff
@@ -0,0 +1,104 @@
+package no.difi.meldingsutveksling.nextmove;
+
+import lombok.extern.slf4j.Slf4j;
+import no.difi.meldingsutveksling.IntegrasjonspunktNokkel;
+import no.difi.meldingsutveksling.MimeTypeExtensionMapper;
+import no.difi.meldingsutveksling.ServiceIdentifier;
+import no.difi.meldingsutveksling.dokumentpakking.service.CmsUtil;
+import no.difi.meldingsutveksling.dokumentpakking.service.CreateAsice;
+import no.difi.meldingsutveksling.domain.MeldingsUtvekslingRuntimeException;
+import no.difi.meldingsutveksling.domain.NextMoveStreamedFile;
+import no.difi.meldingsutveksling.domain.StreamedFile;
+import no.difi.meldingsutveksling.nextmove.message.CryptoMessagePersister;
+import no.difi.meldingsutveksling.nextmove.message.FileEntryStream;
+import no.difi.meldingsutveksling.noarkexchange.MessageContext;
+import no.difi.meldingsutveksling.noarkexchange.StatusMessage;
+import no.difi.meldingsutveksling.pipes.Plumber;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.PipedOutputStream;
+import java.security.cert.X509Certificate;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static no.difi.meldingsutveksling.ServiceIdentifier.DPE;
+
+@Component
+@Slf4j
+public class AsicHandler {
+
+    private final IntegrasjonspunktNokkel keyHelper;
+    private final CryptoMessagePersister cryptoMessagePersister;
+    private final Plumber plumber;
+
+    public AsicHandler(IntegrasjonspunktNokkel keyHelper, CryptoMessagePersister cryptoMessagePersister, Plumber plumber) {
+        this.keyHelper = keyHelper;
+        this.cryptoMessagePersister = cryptoMessagePersister;
+        this.plumber = plumber;
+    }
+
+    InputStream createEncryptedAsic(NextMoveOutMessage msg, MessageContext messageContext) {
+
+        if (msg.getFiles() == null || msg.getFiles().isEmpty()) return null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00122)